### PR TITLE
[2.0] Split and move doc to own folder

### PR DESCRIPTION
**Pull request by magnetik**
Hi,

Let's try to move forward for an awesome 2.0 release, so it's part of #381

I only splitted everything and removed the "Only in Test Environment" that seemed not useful to me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,796 +5,22 @@
...
-
-- [changelog](CHANGELOG.md)
-- [Installation](#installation)
-- [Basic usage and methods](#basic-usage)
-- [Command tests](#command-tests)
-- [Database and fixtures](#database-tests)
-- [Create an already logged client](#create-an-already-logged-client)
-- [Query Counter](#query-counter)
-- [Caveats](#caveats)
-- [paratest](paratest.md)
```
Comment (alexislefebvre) [inline README.md]: It looks like links to `paratest.md` and `fastest.md` are missing. Could you please restore them?

Comment (magnetik) [reply README.md]: oh right, I need to move them too. Will do

File: README.md [hunk 2]
```diff
@@ -5,796 +5,26 @@
...
-    * @IgnoreAnnotation("depends")
-    */
-   class DemoTest extends WebTestCase
-   {
-       // ...
-   }
-   ```
+To run tests in parallel:
+* [Paratest](doc/paratest.md)
+* [Fatests](doc/fastest.md)
```
Comment (alexislefebvre) [inline README.md]: There's a typo here in *Fatests*. :wink:

**Comment by alexislefebvre** [comment]
Thanks a lot @magnetik!

**Pull request merged**